test/e2e/fixtures/given.go

```diff
@@ -0,0 +1,63 @@
+package fixtures
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"sigs.k8s.io/yaml"
+
+	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+	"github.com/argoproj/argo/pkg/client/clientset/versioned/typed/workflow/v1alpha1"
+)
+
+type Given struct {
+	t          *testing.T
+	client     v1alpha1.WorkflowInterface
+	wf         *wfv1.Workflow
+}
+
+// creates a workflow based on the parameter, this may be:
+//
+// 1. A file name if it starts with "@"
+// 2. Raw YAML.
+func (g *Given) Workflow(text string) *Given {
```

Comment:
Very minor, but wouldn't it be clearer to have separate methods instead of using `@`? E.g. `Workflow` and `WorkflowFile`

Reply:
I wanted something that similar to curl. I don't feel strongly about it. @sarabala1979 ?